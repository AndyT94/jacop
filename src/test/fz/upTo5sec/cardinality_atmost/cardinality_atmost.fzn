var 2..2: nvar:: output_var;
array [1..5] of var int: variables:: output_array([1..5]) = [2,1,7,1,2];
array [1..4] of var int: values:: output_array([1..4]) = [5,7,2,9];
solve  satisfy;
