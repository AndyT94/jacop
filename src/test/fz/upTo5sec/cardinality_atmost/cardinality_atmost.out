nvar = 2;
variables = array1d(1..5, [2, 1, 7, 1, 2]);
values = array1d(1..4, [5, 7, 2, 9]);
----------